src/code/middleware/its-log.js

```diff
@@ -162,7 +163,7 @@ function createLogEntry(loggingMetadata, action, nextState){
   delete context.type;
   delete context.meta;
 
-  context = makeMutable(context);
+  context = makeMutable(cloneDeep(context));
```

Comment:
I don't quite understand. You were seeing the strings `"male"` and `"female"` after `makeMutable`? Can't see how that would have happened without diving deep into the `makeMutable` function.